contracts/Oracle.sol

```diff
@@ -159,15 +165,22 @@ contract Oracle is Ownable {
      * @return True if dispute period is over, False if not
      */
     function isDisputePeriodOver(address _asset, uint256 _expiryTimestamp) public view returns (bool) {
-        // check if the pricer has a price for this expiry timestamp
-        Price memory price = storedPrice[_asset][_expiryTimestamp];
-        if (price.timestamp == 0) {
-            return false;
+        uint256 price = stablePrice[_asset];
```

Comment:
this is good, just wondering, can we accidentally end up with having an asset with a pricer and have it as a stableAsset?

Reply:
do we need a check when setting the pricer / setting it as stable?

Reply:
mm yeah, e.g if we have a pricer for WETH and set accidentally a stable price for WETH, then we will always return WETH price + no locking period + no dispute period.

Reply:
Good idea, I will add in another PR a check for that.

Reply:
#311